src/commons/bgp/iptree.rs

```diff
@@ -2,9 +2,9 @@ use std::{collections::HashMap, ops::Range};
 
 use intervaltree::IntervalTree;
 
-use rpki::repository::resources::ResourceSet;
+use rpki::repository::resources::{Addr, AddressRange, ResourceSet};
 
-use crate::commons::api::TypedPrefix;
+use crate::commons::api::{Ipv4Prefix, Ipv6Prefix, TypedPrefix};
 
 //------------ IpRange -----------------------------------------------------
```

Comment:
Would it make sense to clean up the range code in rpki-rs further and then use its types directly?

Reply:
For that there would first need to be a V4/V6 range in rpki-rs (which strikes me as a good idea). However, the Krill counterparts are used far and wide throughout the system and _slightly_ differ implementation-wise, so I am not sure if we want to do that as part of this PR.